app/src/main/java/it/netknights/piauthenticator/viewcontroller/MainActivity.java

```diff
@@ -191,8 +192,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
     void pushAuthRequestReceived(Intent intent) {
         // intent contain push auth info
-        logprint("Intent Found onCreate:");
-        logprint(intent.getExtras().toString());
+        logprint("Intent found onCreate: " + ((intent.getExtras() != null) ? intent.getExtras().toString() : ""));
```

Comment:
In my opinion this should clearly state that there are no extras. This could make debugging easier if it ever turns out to be a problem.